EzMap.Domain/Models/PoiCollectionConfiguration.cs

```diff
@@ -0,0 +1,15 @@
+﻿using Microsoft.EntityFrameworkCore;
+using Microsoft.EntityFrameworkCore.Metadata.Builders;
+
+namespace EzMap.Domain.Models;
+
+public class PoiCollectionConfiguration : IEntityTypeConfiguration<PoiCollection>
+{
+    public void Configure(EntityTypeBuilder<PoiCollection> builder)
```

Comment:
Configuration with Property (which is not really necessary, it's there on the class by default) but without Relationship (which is a lot more important) is really weird.

Again it's about what you want to express (what you want other developers to see straightaway looking at your code), they pretty much want to see the relationship (which when configured by convention is much harder to spot than just a pure property) explicitly configured. Do they want to see a property configuration which does not inform anything new? Probably not.

Reply:
This is minor, and usually I wouldn't scrutinize on this, but in general this set a bad example for the overall repo, and for a small repo like this, building (& learning conventions) are first priority not really what functionality is implemented.